packages/client/src/Controllers/CanvasController.tsx

```diff
@@ -0,0 +1,109 @@
+// Отрисовка игрового поля
+const drawField = (ctx: CanvasRenderingContext2D, board: number[][], bg: HTMLImageElement): void => {
```

Comment:
Будет полезным вынести отдельные типы для доски и ячейчки, потому что они много где переиспользуются

```suggestion
const drawField = (ctx: CanvasRenderingContext2D, board: Board, bg: HTMLImageElement): void => {
```